packages/google_maps_routes_api/analysis_options.yaml

```diff
@@ -0,0 +1,7 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
These analysis files are also something that are not used in separate plugins.

Reply:
Note that this is not a Flutter plugin. Does the same apply to packages as well? I can see that there are several packages in this repository which have their own analysis_options with custom rules.

Reply:
Okay, then these are little bit different.